components/ModSubmissionListContainer.vue

```diff
@@ -32,11 +32,16 @@
                     class="grid grid-cols-subgrid col-span-5 bg-gray-200 cursor-pointer hover:bg-primary"
                     @click="handleClickToSubmissionForm(submission.id)"
                 >
-                    <span class="text-start">{{ index + 1 }}</span>
-                    <span class="text-start">{{ submission.healthcareProfessionalName }}</span>
-                    <span class="text-start">{{ getSubmissionStatus(submission) }}</span>
-                    <span class="text-start">{{ convertDateToLocalTime(submission.updatedDate) }}</span>
-                    <span class="text-start">{{ convertDateToLocalTime(submission.createdDate) }}</span>
+                    <nuxt-link
+                        :to="`/moderation/editsubmission/${submission.id}`"
+                        class="grid grid-cols-subgrid col-span-5 bg-gray-200 p-1 hover:bg-primary"
```

Comment:
let's use our theme colors

```suggestion
                        class="grid grid-cols-subgrid col-span-5 bg-primary-text-muted p-1 hover:bg-primary"
```

Reply:
Will do! Sometimes I don't notice the tailwind when just changing the functionality. I'll try to look more at the files I am working on in order to get rid of the old styling